src/string.js

```diff
@@ -137,8 +147,8 @@ inherits(StringSchema, MixedSchema, {
   },
 
   uppercase(message = locale.uppercase) {
-    return this.transform(
-      value => (!isAbsent(value) ? value.toUpperCase() : value),
+    return this.transform(value =>
```

Comment:
you shouldnt by any means force your code style onto the code style for the one you're contributing to, comply with the author code style and try again